lib/pbench/server/indexer.py

```diff
@@ -134,11 +130,14 @@ def get_es(config):
     hosts = _get_es_hosts(config)
     if hosts is None:
         return None
+
     # FIXME: we should just change these two loggers to write to a
     # file instead of setting the logging level up so high.
     logging.getLogger("urllib3").setLevel(logging.FATAL)
     logging.getLogger("elasticsearch1").setLevel(logging.FATAL)
-    return Elasticsearch(hosts, max_retries=0)
+
+    timeoutobj = Timeout(total=1200, connect=10, read=_read_timeout)
+    return Elasticsearch(hosts, max_retries=0, timeout=timeoutobj)
```

Comment:
I tried to figure out whether the code accepted a constructor kwarg for timeout outside the host `dict`s ... I'm glad you found it. (Assuming you didn't just make it up? 😆 )

Reply:
I just made it up...but, I think it actually might work!  😁

Reply:
I'm sure we **_can_**...I'm equally sure that we don't currently, so I'm assuming that this change doesn't make anything _worse_....  😇